src/main/java/pl/project13/jgit/JGitCommon.java

```diff
@@ -63,38 +61,32 @@ public JGitCommon(LoggerBridge log) {
   }
 
   private Collection<String> getTags(final Git git, final ObjectId objectId, final RevWalk finalWalk) throws GitAPIException {
-    List<Ref> tagRefs = git.tagList().call();
-    Collection<Ref> tagsForHeadCommit = Collections2.filter(tagRefs, new Predicate<Ref>() {
-      @Override
-      public boolean apply(Ref tagRef) {
-        try {
-          final RevCommit tagCommit = finalWalk.parseCommit(tagRef.getObjectId());
-          final RevCommit objectCommit = finalWalk.parseCommit(objectId);
-          if (finalWalk.isMergedInto(objectCommit, tagCommit)) {
-            return true;
-          }
-        } catch (Exception ignored) {
-          log.debug("Failed while getTags [{}] -- ", tagRef, ignored);
-        }
-        return false;
-      }
-    });
-    Collection<String> tags = Collections2.transform(tagsForHeadCommit, new Function<Ref, String>() {
-      @Override public String apply(Ref input) {
-        return input.getName().replaceAll("refs/tags/", "");
-      }
-    });
-    return tags;
+    return git.tagList().call()
+            .stream()
+            .filter(tagRef -> {
+              try {
+                final RevCommit tagCommit = finalWalk.parseCommit(tagRef.getObjectId());
+                final RevCommit objectCommit = finalWalk.parseCommit(objectId);
+                if (finalWalk.isMergedInto(objectCommit, tagCommit)) {
+                  return true;
+                }
+              } catch (Exception ignored) {
+                log.debug("Failed while getTags [{}] -- ", tagRef, ignored);
+              }
+              return false;
+            })
+            .map(tagRef -> tagRef.getName().replaceAll("refs/tags/", ""))
```

Comment:
Why not use ``trimFullTagName(tagRef.getName())`` here?

This specific Method would use ``replaceFirst("refs/tags/", "")`` instead of ``replaceAll``. However AFAIK we are expecting only the start to be ``refs/tags/...``. I think a ``replaceAll`` would prevent us from using actual tags that are looking like ``refs/tags/...`` (very unlikely that this ever happens). In practice I wouldn't assume it actually makes a difference.